### PR TITLE
TV weights equal or less than zero

### PR DESCRIPTION
**Pull request by tjlane**
This PR resolves issue #26 and implements a new `.from_numpy_map` method on `Map`. The latter lets one instantiate a map object from a numpy array in one step instead of two.

Happy to have suggestions of how to name the function to make it clear that the numpy array should be a 3-d array of voxels in real space...

### CODE REVIEW

**Comment by tjlane** [comment]
+ fix bug where `tv` methods were not propagating (unused) `uncertainties` from the input to output

File: meteor/rsmap.py [hunk 1]
```diff
@@ -308,6 +309,23 @@ def from_gemmi(
             uncertainty_column=uncertainty_column,
         )
 
+    @classmethod
```
Comment (alisiafadini) [inline meteor/rsmap.py:312]: docstring??

File: meteor/rsmap.py [hunk 2]
```diff
@@ -308,6 +309,23 @@ def from_gemmi(
...
+    def from_numpy_map(
+        cls, map_grid: np.ndarray, *, spacegroup: Any, cell: Any, high_resolution_limit: float
+    ) -> Map:
```
Comment (alisiafadini) [inline meteor/rsmap.py:315]: re: your comment in PR

I think even just `from_3Dnumpy_to_rsmap` would go a long way..

File: meteor/tv.py [hunk 3]
```diff
@@ -29,6 +26,8 @@ class TvDenoiseResult:
 
 def _tv_denoise_array(*, map_as_array: np.ndarray, weight: float) -> np.ndarray:
     """Closure convienence function to generate more readable code."""
+    if weight <= 0.0:
```
Comment (alisiafadini) [inline meteor/tv.py:29]: smooth :)

File: test/unit/test_tv.py [hunk 4]
```diff
@@ -44,10 +44,22 @@ def test_tv_denoise_map_smoke(
...
         assert isinstance(output[1], tv.TvDenoiseResult)
     else:
-        assert isinstance(output, rs.DataSet)
+        assert isinstance(output, Map)
+
+
+def test_tv_denoise_zero_weight(random_difference_map: Map) -> None:
+    weight = 0.0
+    output = tv.tv_denoise_difference_map(
+        random_difference_map,
```
Comment (alisiafadini) [inline test/unit/test_tv.py:56]: capital :)

**Review by alisiafadini — APPROVED** [review]

**Pull request merged**